docs/resources/client/index.md

```diff
@@ -0,0 +1,15 @@
+Resource Packs
+==============
+
+[Resource Packs][respack] allow for the customization of client resources through the `assets` directory. This includes textures, models, sounds, localizations, etc. Forge, and your mod, can also have resource packs. Any user can therefore modify all the textures, models, and other asetts defined within this directory.
```

Comment:
It may be just me, but the use of `etc.` at the end of a sentence made me join that sentence and the next one together. 

We want to give emphasis to mods being capable of holding resource packs, with Forge as the sidenote. Also, a typo:
```suggestion
[Resource Packs][respack] allow for the customization of client resources through the `assets` directory. This includes textures, models, sounds, localizations, and others. Your mod (as well as Forge itself) can also have resource packs. Any user can therefore modify all the textures, models, and other assets defined within this directory.
```